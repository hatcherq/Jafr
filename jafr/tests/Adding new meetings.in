2
Go to school
24/05/23
13:00



