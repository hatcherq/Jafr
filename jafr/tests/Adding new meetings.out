Please enter a meeting description:

Please enter a date:

Please enter a time:

Ok, I have added Go to school on 24/05/23 at 13:00.
Would you like to share this meeting? [y/n]: 



